### PR TITLE
add build version

### PR DESCRIPTION
**Pull request by oliviazhang0809**

### CODE REVIEW

**Comment by oliviazhang0809** [comment]
@msv @gyeh ^^

File: http/src/main/scala/com/paypal/cascade/http/server/StatusResponse.scala [hunk 1]
```diff
@@ -52,6 +58,7 @@ object StatusResponse extends LoggingSugar {
    * @return a StatusResponse object
    */
   def getStatusResponse(props: BuildProperties, serviceName: String): StatusResponse = {
+    val buildVersion = props.get("service.buildVersion").getOrElse("VersionNotSet")
```
Comment (msv) [inline http/src/main/scala/com/paypal/cascade/http/server/StatusResponse.scala]: @oliviazhang0809 what about VersionNotFound? if it's not in props I don't know if that's an indication that's not set.

File: http/src/main/scala/com/paypal/cascade/http/server/StatusResponse.scala [hunk 2]
```diff
@@ -15,21 +15,27 @@
...
+ * @param buildVersion build version of current service
  * @param dependencies the dependencies used
  * @param gitInfo the repo info for this build
  */
 case class StatusResponse(
+  @JsonProperty("status")
   status: String,
   @JsonProperty("service-name")
   serviceName: String,
+  @JsonProperty("build-version")
```
Comment (msv) [inline http/src/main/scala/com/paypal/cascade/http/server/StatusResponse.scala]: @oliviazhang0809 could this ever be confused with jenkins/CI build versions? what about service-version or something similar?

Comment (oliviazhang0809) [inline http/src/main/scala/com/paypal/cascade/http/server/StatusResponse.scala]: @msv yeah i like service-version

**Comment by oliviazhang0809** [comment]
Not sure why it failed. `sbt clean test` is successful locally.

**Comment by msv** [comment]
:shipit:

**Pull request merged**